protoc-gen-twirp/generator.go

```diff
@@ -114,22 +114,42 @@ func (t *twirp) Generate(in *plugin.CodeGeneratorRequest) *plugin.CodeGeneratorR
 	}
 	t.genPkgName = genPkgName
 
+	// We also need to figure out the fully import path of the package we're
```

Comment:
The comments help tremendously with understanding the nuance of how package names are assigned -- thanks!